x/ccv/provider/keeper/partial_set_security.go

```diff
@@ -110,18 +110,18 @@ func (k Keeper) ComputeMinPowerToOptIn(ctx sdk.Context, chainID string, bondedVa
 	for _, val := range bondedValidators {
 		power := k.stakingKeeper.GetLastValidatorPower(ctx, val.GetOperator())
 		powers = append(powers, power)
-		totalPower = totalPower.Add(sdk.NewDecFromInt(sdk.NewInt(power)))
+		totalPower = totalPower.Add(sdk.NewDec(power))
 	}
 
 	// sort by powers descending
 	sort.Slice(powers, func(i, j int) bool {
 		return powers[i] > powers[j]
 	})
 
-	topNThreshold := sdk.NewDecFromInt(sdk.NewInt(int64(topN))).QuoInt64(int64(100))
+	topNThreshold := sdk.NewDec(int64(topN)).QuoInt64(int64(100))
 	powerSum := sdk.ZeroDec()
 	for _, power := range powers {
-		powerSum = powerSum.Add(sdk.NewDecFromInt(sdk.NewInt(power)))
+		powerSum = powerSum.Add(sdk.NewDec(power))
```

Comment:
Using directly `NewDec` here and above because of "Code Efficiency Improvements and Optimization Opportunities" suggestions.